core-plugins/src/e2e-test/features/errorcollector/ErrorCollector.feature

```diff
@@ -71,3 +71,110 @@ Feature: Error Collector - Verification of successful error emitted to ErrorColl
     Then Verify the pipeline status is "Succeeded"
     Then Verify the CSV Output File matches the Expected Output File: "errorCollectorOutput" With Expected Partitions: "expectedErrorCollectorOutputPartitions"
     Then Close the pipeline logs
+
+  @FILE_SOURCE_TEST @FILE_SINK_TEST @ErrorCollector
+  Scenario: Verify error collector plugin functionality with default error collector config using File source to Wrangler to ErrorCollector to File sink pipeline
```

Comment:
nit - `Verify error collector plugin functionality with default config using File Plugins and Wrangler`

Reply:
Changed description